openspoor/visualisations/trackmap.py

```diff
@@ -165,12 +166,17 @@ def __init__(self, df, lat_column: str = 'lat', lon_column: str = 'lon', popup:
         :param url_column: A column including an url that is displayed in the popup
         """
 
-        super().__init__(df)
-        # TODO: Make this a helper function? Also required for linestring dataframe parsing
+        if not isinstance(df, pd.DataFrame):
+            df = pd.DataFrame(df)
```

Comment:
Interessant dit, waarom is dit nodig? :P

Reply:
Dit is voor als je het volgende doet:
PlottingDataFrame({'a': [1,2])
. Hier herkent hij niet dat de dict die je meegeeft eigenlijk het eerste argument is (data= ) . 
Vergelijk dit met
pd.DataFrame({'a': [1,2])
Waar hij dit wel begrijpt. Van te voren de df omzetten naar een dataframe laat de bovenste syntax dus toe (en ik kan geen gevallen bedenken waarin het kwaad kan).
Dit stukje voorbewerking ben ik niet heel tevreden over, maar tot nog toe doet het wat het moet doen.